deploy/docker/math.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM docker.io/clojure:tools-deps
+FROM --platform=linux/amd64 docker.io/clojure:tools-deps-1.11.1.1208-alpine
```

Comment:
You may have done this because the default ubuntu base has become problematic (I ran into this recently). I have a preference for sticking with a debian base, for familiarity sake as much as anything, but this is something we can discuss.

Reply:
Yes its because Debian and Ubuntu both have a huge amount of dependencies that are considered vulnerable in our governance tests - I'd like to keep our production system using Alpine becuase of this minimal approach.